docs/source/release_notes/release-notes.rst

```diff
@@ -42,34 +42,39 @@ develop
          - Change
 
     *    - |new|
+         - ConnectionSupplier consumers can now choose to receive a brand new unshared connection.
```

Comment:
On its own, this doesn't mean much to me. I suggest prefixing it with `OracleKVS:` (or DbKvs if it works for both that and postgres).